### PR TITLE
build: use release-please method and doc it

### PR DESCRIPTION
**Pull request by Tal500**
Additionally, remove the `standard-version` package.

This is an important docs in README.md, explaining how to make a new release via release-please and a manual invocation.

### CODE REVIEW

**Comment by orefalo** [comment]
nice to see the switch to pnpm, it's a great pkg mgr.
I use it for a monorepo of mine, a large project I am working on. it's neat

File: package.json [hunk 1]
```diff
@@ -37,8 +37,7 @@
 		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. src/ && eslint --ignore-path .gitignore src/",
 		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. src/",
 		"commit-preverify": "npx lint-staged && npm run check",
-		"commit": "npm run commit-preverify && cross-env NO_VERIFY=1 npx cz",
-		"release": "standard-version --no-verify"
+		"commit": "npm run commit-preverify && cross-env NO_VERIFY=1 npx cz"
```
Comment (orefalo) [inline package.json:40]: this should be..
pnpm run

Comment (orefalo) [reply package.json:40]: there is also pnpx, as a substitute to npx

Comment (orefalo) [reply package.json:40]: and if you agree @Tal500, I can add a few tags in package.json to for pnpm only.  let me know

Comment (Tal500) [reply package.json:40]: > this should be.. pnpm run

I can, I just want it to be resilient to a weak environment(e.g. automated scripting/github actions, etc.), in the case pnpm isn't installed.

Comment (Tal500) [reply package.json:40]: If you really want you may change them, just wanted to be "safe" or "portable"

Comment (orefalo) [reply package.json:40]: it's fine as is,
 I forgot about the actions, you are right.

**Pull request merged**